aleph/tests/test_view_util.py

```diff
@@ -32,3 +35,33 @@ def test_sanitize_html(self):
         assert attr == "https://example.org/blockchain", html
         assert html.find(".//a").get("target") == "_blank", html
         assert "nofollow" in html.find(".//a").get("rel"), html
+
+    def test_validate_returns_errors_for_paths(self):
+        # given
+        schema = "RoleCreate" # name min length 4, password min length 6
+        data = json.loads('{"name":"Bob","password":"1234","code":"token"}')
+
+        # then
+        with self.assertRaises(BadRequest):
+            validate(data, schema)
+
+        # and
```

Comment:
Don't know how to check that error was raised and verify error details in one run 🤔

Reply:
@sunu thanks, that worked!